frontend/simple/views/Home.vue

```diff
@@ -33,10 +33,7 @@
 <style scoped lang="scss">
 @import "../assets/sass/theme/index";
 
-.container {
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
+.splash {
   justify-content: center;
```

Comment:
Hmm, since this deletes `display: flex`, it's no longer using flexbox?

Reply:
`.gi-routerView` (which gets set when there's a sidebar) sets flex: https://github.com/okTurtles/group-income-simple/blob/master/frontend/simple/assets/sass/_groupincome.scss#L34